docs/data/data-grid/api-object/api-object.md

```diff
@@ -1,8 +1,4 @@
----
-title: Data Grid - API object
----
```

Comment:
Why removing this meta data which is present in all the other files?

Reply:
It duplicates with the h1 now. It's the same value.

Reply:
Sounds like it's always the same value except for pro/premium features which have the icon the `<h1>` and not in the title

Reply:
The capitalization is different in most of the places we're we do this, e.g. https://github.com/mui/mui-x/blob/f1294f981c78143b4670ef20276416f6d40946ed/docs/data/data-grid/column-definition/column-definition.md?plain=1#L1-L5

There is not duplication. However, based on @samuelsycamore https://www.notion.so/mui-org/Style-conventions-for-MUI-components-and-products-dff7d2d7d1ba4a4abd47f48cf345b800#e1ef8fa1b88d4f27819604f6ffd6f537, we are replacing the `<h1>` to match the `<title>`, to use the Title Case convention too.

Reply:
To be very clear, on the line that you comment on, I did

```diff
-# Data grid - API object
+# Data Grid - API object
```